### DISCUSSION TITLE
New dialogs for lights, switches and siren entities- Configuration to override this on entities card

### DISCUSSION THREAD

**Discussion by imonitormyhome** [home-assistant/frontend#16115] [Dashboard]
I hate this new feature of seeing switches when I click on a switch on my entity card, totally dislike it so much I rolled back to 12-2022. 
I really need a way to disable it and bring display the history (SINGLE CLICK) instead of the dreaded Switch.

**Comment by z3liff** [comment]
agreed.  I would love to see the older dialogs as an option.  On switches with effects/colors/ etc., the new layout cripples usability.

**Comment by ildar170975** [comment]
+1
